<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Ajouter_Employe</class>
 <widget class="QDialog" name="Ajouter_Employe">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>873</width>
    <height>484</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Admin Pannel</string>
  </property>
  <property name="windowIcon">
   <iconset resource="MyResources.qrc">
    <normaloff>:/new/prefix1/MyResources/kissclipart-login-png-clipart-user-computer-icons-a59899d3a10c7dde.png</normaloff>:/new/prefix1/MyResources/kissclipart-login-png-clipart-user-computer-icons-a59899d3a10c7dde.png</iconset>
  </property>
  <property name="styleSheet">
   <string notr="true">QDialog#Ajouter_Employe{
border-image:url(:/new/prefix1/MyResources/39496316340_66986bb280_b.jpg);}
</string>
  </property>
  <widget class="QTabWidget" name="tabWidget">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>881</width>
     <height>491</height>
    </rect>
   </property>
   <property name="accessibleName">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;QTabWidget#tabWidget{&lt;/p&gt;&lt;p&gt;background-image: url(:/new/prefix1/MyResources/80745_c977e1f4028f918bae1e810a47e3c920.png);&lt;/p&gt;&lt;p&gt;}&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
   <property name="autoFillBackground">
    <bool>false</bool>
   </property>
   <property name="styleSheet">
    <string notr="true">#tabWidget{
border-image: url(:/new/prefix1/MyResources/59524298_595553970930082_8770854359857627136_n.jpg);
background-color: rgba(0, 0, 0, 0%);}</string>
   </property>
   <property name="currentIndex">
    <number>1</number>
   </property>
   <widget class="QWidget" name="Ajt_Emp">
    <property name="styleSheet">
     <string notr="true">#Ajt_Emp{
border-image: url(:/new/prefix1/MyResources/59524298_595553970930082_8770854359857627136_n.jpg);
background-color: rgba(0, 0, 0, 0%);}</string>
    </property>
    <attribute name="title">
     <string>Ajouter compte</string>
    </attribute>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>190</y>
       <width>101</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>16</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Login ID :</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>230</y>
       <width>171</width>
       <height>51</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>16</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Login password :</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_3">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>140</y>
       <width>141</width>
       <height>41</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>16</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Employe id :</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_Emp_id">
     <property name="geometry">
      <rect>
       <x>180</x>
       <y>150</y>
       <width>151</width>
       <height>20</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">  QLineEdit {
    border-style: outset;
    border-width: 2px;
    border-radius: 10px;
    border-color: beige;
    font: bold 7px;
    min-width: 5em;
    padding:3px;
background-color: rgba(0, 0, 0, 50);
}
</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_loginid">
     <property name="geometry">
      <rect>
       <x>180</x>
       <y>200</y>
       <width>151</width>
       <height>20</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">  QLineEdit {
    border-style: outset;
    border-width: 2px;
    border-radius: 10px;
    border-color: beige;
    font: bold 7px;
    min-width: 5em;
    padding:3px;
background-color: rgba(0, 0, 0, 50);
}</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_loginpass">
     <property name="geometry">
      <rect>
       <x>180</x>
       <y>250</y>
       <width>151</width>
       <height>20</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">  QLineEdit {
    border-style: outset;
    border-width: 2px;
    border-radius: 10px;
    border-color: beige;
    font: bold 7px;
    min-width: 5em;
    padding:3px;
background-color: rgba(0, 0, 0, 50);
}</string>
     </property>
    </widget>
    <widget class="QTableView" name="table_empl">
     <property name="geometry">
      <rect>
       <x>380</x>
       <y>100</y>
       <width>481</width>
       <height>241</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QTableView {
background: rgba(255, 255, 255,50);
}
QTableView QTableCornerButton::section
{
     background: rgba(0, 0, 0, 0%);
}
QTableView::item:focus

{
    color:white;
}
QTableView::item:selected
{
    color:black;
     background: rgba(0, 0, 0, 50%);
}</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_4">
     <property name="geometry">
      <rect>
       <x>380</x>
       <y>40</y>
       <width>211</width>
       <height>41</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>16</pointsize>
      </font>
     </property>
     <property name="text">
      <string>List des employes : </string>
     </property>
    </widget>
    <widget class="QWidget" name="layoutWidget">
     <property name="geometry">
      <rect>
       <x>480</x>
       <y>410</y>
       <width>381</width>
       <height>41</height>
      </rect>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_3">
      <item>
       <widget class="QCommandLinkButton" name="commandLinkButton">
        <property name="text">
         <string>Ajouter </string>
        </property>
        <property name="icon">
         <iconset resource="MyResources.qrc">
          <normaloff>:/new/prefix1/MyResources/newajouter.png</normaloff>:/new/prefix1/MyResources/newajouter.png</iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>25</width>
          <height>25</height>
         </size>
        </property>
        <property name="checkable">
         <bool>true</bool>
        </property>
        <property name="checked">
         <bool>false</bool>
        </property>
        <property name="autoRepeat">
         <bool>false</bool>
        </property>
        <property name="autoExclusive">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCommandLinkButton" name="commandLinkButton_2">
        <property name="text">
         <string>Disconnect</string>
        </property>
        <property name="icon">
         <iconset resource="MyResources.qrc">
          <normaloff>:/new/prefix1/MyResources/logout-icon.png</normaloff>:/new/prefix1/MyResources/logout-icon.png</iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>25</width>
          <height>25</height>
         </size>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
    <widget class="QPushButton" name="pushButton_Trier">
     <property name="geometry">
      <rect>
       <x>780</x>
       <y>350</y>
       <width>75</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="icon">
      <iconset resource="MyResources.qrc">
       <normaloff>:/new/prefix1/MyResources/trier1063846914-1024x1024.png</normaloff>:/new/prefix1/MyResources/trier1063846914-1024x1024.png</iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>30</width>
       <height>30</height>
      </size>
     </property>
     <property name="flat">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QToolButton" name="Settings">
     <property name="geometry">
      <rect>
       <x>804</x>
       <y>10</y>
       <width>61</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>Settings</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="Supp_Emp">
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="styleSheet">
     <string notr="true">#Supp_Emp{
border-image: url(:/new/prefix1/MyResources/59524298_595553970930082_8770854359857627136_n.jpg);
background-color: rgba(0, 0, 0, 0%);}</string>
    </property>
    <attribute name="title">
     <string>Supprimer  compte</string>
    </attribute>
    <widget class="QTableView" name="tableSupp">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>180</y>
       <width>791</width>
       <height>192</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QTableView {
background: rgba(255, 255, 255,50);
}
QTableView QTableCornerButton::section
{
     background: rgba(0, 0, 0, 0%);
}
QTableView::item:focus

{
    color:white;
}
QTableView::item:selected
{
    color:black;
     background: rgba(0, 0, 0, 50%);
}</string>
     </property>
    </widget>
    <widget class="QPushButton" name="search_bf_delete">
     <property name="geometry">
      <rect>
       <x>740</x>
       <y>80</y>
       <width>91</width>
       <height>51</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="icon">
      <iconset resource="MyResources.qrc">
       <normaloff>:/new/prefix1/MyResources/icons8-search-512.png</normaloff>:/new/prefix1/MyResources/icons8-search-512.png</iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>30</width>
       <height>30</height>
      </size>
     </property>
     <property name="flat">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QWidget" name="layoutWidget">
     <property name="geometry">
      <rect>
       <x>560</x>
       <y>40</y>
       <width>251</width>
       <height>31</height>
      </rect>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QLabel" name="label_11">
        <property name="font">
         <font>
          <pointsize>16</pointsize>
         </font>
        </property>
        <property name="text">
         <string>Login ID :</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="lineEdit_Supp">
        <property name="styleSheet">
         <string notr="true">  QLineEdit {
    border-style: outset;
    border-width: 2px;
    border-radius: 10px;
    border-color: beige;
    font: bold 7px;
    min-width: 5em;
    padding:3px;
background-color: rgba(0, 0, 0, 50);
}</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
    <widget class="QWidget" name="layoutWidget">
     <property name="geometry">
      <rect>
       <x>470</x>
       <y>410</y>
       <width>377</width>
       <height>57</height>
      </rect>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_4">
      <item>
       <widget class="QCommandLinkButton" name="pb_supp">
        <property name="text">
         <string>Supprimer </string>
        </property>
        <property name="icon">
         <iconset resource="MyResources.qrc">
          <normaloff>:/new/prefix1/MyResources/delete.png</normaloff>
          <normalon>:/new/prefix1/MyResources/deleteClicked.png</normalon>
          <disabledoff>:/new/prefix1/MyResources/delete.png</disabledoff>
          <disabledon>:/new/prefix1/MyResources/deleteClicked.png</disabledon>
          <selectedoff>:/new/prefix1/MyResources/delete.png</selectedoff>
          <selectedon>:/new/prefix1/MyResources/deleteClicked.png</selectedon>:/new/prefix1/MyResources/delete.png</iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>25</width>
          <height>25</height>
         </size>
        </property>
        <property name="checkable">
         <bool>true</bool>
        </property>
        <property name="checked">
         <bool>false</bool>
        </property>
        <property name="autoRepeat">
         <bool>true</bool>
        </property>
        <property name="autoExclusive">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCommandLinkButton" name="commandLinkButton_disconnect">
        <property name="text">
         <string>Disconnect</string>
        </property>
        <property name="icon">
         <iconset resource="MyResources.qrc">
          <normaloff>:/new/prefix1/MyResources/logout-icon.png</normaloff>:/new/prefix1/MyResources/logout-icon.png</iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>25</width>
          <height>25</height>
         </size>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </widget>
   <widget class="QWidget" name="Aff_Emp">
    <property name="styleSheet">
     <string notr="true">#Aff_Emp{
border-image: url(:/new/prefix1/MyResources/59524298_595553970930082_8770854359857627136_n.jpg);
background-color: rgba(0, 0, 0, 0%);}</string>
    </property>
    <attribute name="title">
     <string>Afficher  compte</string>
    </attribute>
    <widget class="QTableView" name="tableView">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>230</y>
       <width>801</width>
       <height>181</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QTableView {
background: rgba(255, 255, 255,50);
}
QTableView QTableCornerButton::section
{
     background: rgba(0, 0, 0, 0%);
}
QTableView::item:focus

{
    color:white;
}
QTableView::item:selected
{
    color:black;
     background: rgba(0, 0, 0, 50%);
}</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton">
     <property name="geometry">
      <rect>
       <x>320</x>
       <y>190</y>
       <width>91</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="icon">
      <iconset resource="MyResources.qrc">
       <normaloff>:/new/prefix1/MyResources/icons8-search-512.png</normaloff>:/new/prefix1/MyResources/icons8-search-512.png</iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>30</width>
       <height>30</height>
      </size>
     </property>
     <property name="flat">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QWidget" name="layoutWidget">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>30</y>
       <width>381</width>
       <height>151</height>
      </rect>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <item>
         <widget class="QLabel" name="label_10">
          <property name="font">
           <font>
            <pointsize>16</pointsize>
           </font>
          </property>
          <property name="text">
           <string>Login ID :              </string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="lineEdit_4">
          <property name="styleSheet">
           <string notr="true">  QLineEdit {
    border-style: outset;
    border-width: 2px;
    border-radius: 10px;
    border-color: beige;
    font: bold 7px;
    min-width: 5em;
    padding:3px;
background-color: rgba(0, 0, 0, 50);
}</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_5">
        <item>
         <widget class="QLabel" name="label_5">
          <property name="font">
           <font>
            <pointsize>16</pointsize>
           </font>
          </property>
          <property name="text">
           <string>Type de recherche  :         </string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QComboBox" name="comboBox_Type_Rc">
          <property name="styleSheet">
           <string notr="true">QComboBox {
    border-style: outset;
    border-width: 2px;
    border-radius: 10px;
    border-color: beige;
    font: MS Shell Dlg 2 7px;
    min-width: 5em;
    padding:1px;
    background-color: rgba(0,0,0,50%);

}
QComboBox QAbstractItemView {
border: 2px solid darkgray;
selection-background-color: lightgray;
color:rgb(87,117,131);
}
selection-color: rgba(0, 0, 0,50%);
selection-background-color: rgba(0, 0, 0, 50);</string>
          </property>
          <item>
           <property name="text">
            <string>Select</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Standard</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>avancée</string>
           </property>
          </item>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_6">
        <item>
         <widget class="QLabel" name="label_6">
          <property name="font">
           <font>
            <pointsize>14</pointsize>
           </font>
          </property>
          <property name="text">
           <string>Ordre :</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QRadioButton" name="radioButton_c">
          <property name="font">
           <font>
            <pointsize>14</pointsize>
           </font>
          </property>
          <property name="text">
           <string>croissant</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QRadioButton" name="radioButton_d">
          <property name="font">
           <font>
            <pointsize>14</pointsize>
           </font>
          </property>
          <property name="text">
           <string>décroissant</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </widget>
  </widget>
 </widget>
 <resources>
  <include location="MyResources.qrc"/>
 </resources>
 <connections/>
</ui>
